src/app/pages/organization/site-areas/list/site-areas-list-table-data-source.ts

```diff
@@ -203,99 +178,90 @@ export class SiteAreasListTableDataSource extends TableDataSource<SiteArea> {
-          headerClass: 'col-15em',
-          class: 'col-15em',
-          sortable: true,
-        },
-        {
-          id: 'createdBy',
-          name: 'users.created_by',
-          formatter: (user: User) => Utils.buildUserFullName(user),
-          headerClass: 'col-15em',
-          class: 'col-15em',
-        },
-        {
-          id: 'lastChangedOn',
-          name: 'users.changed_on',
-          formatter: (lastChangedOn: Date) => this.datePipe.transform(lastChangedOn),
-          headerClass: 'col-15em',
-          class: 'col-15em',
-          sortable: true,
-        },
-        {
-          id: 'lastChangedBy',
-          name: 'users.changed_by',
-          formatter: (user: User) => Utils.buildUserFullName(user),
-          headerClass: 'col-15em',
-          class: 'col-15em',
-        },
+          id: 'smartCharging',
+          name: 'site_areas.smart_charging',
+          headerClass: 'col-10p text-center',
+          class: 'col-10p text-center',
+          formatter: (smartCharging: boolean) => smartCharging ?
+            this.translateService.instant('general.yes') : this.translateService.instant('general.no'),
+        }
       );
     }
     return tableColumnDef;
   }
 
   public buildTableActionsDef(): TableActionDef[] {
     const tableActionsDef = super.buildTableActionsDef();
-    if (this.canCreateSiteArea) {
-      return [
-        new TableCreateSiteAreaAction().getActionDef(),
-        ...tableActionsDef,
-      ];
-    }
-    return tableActionsDef;
+    return [
+      this.createAction,
+      ...tableActionsDef,
+    ];
   }
 
   public buildTableDynamicRowActions(siteArea: SiteArea): TableActionDef[] {
+    const actions: TableActionDef[] = [];
     const openInMaps = new TableOpenInMapsAction().getActionDef();
     // Check if GPS is available
-    let actions: TableActionDef[];
     openInMaps.disabled = !Utils.containsAddressGPSCoordinates(siteArea.address);
-    if (siteArea.issuer) {
-      if (this.canCrudSiteArea &&
-          this.authorizationService.isSiteAdmin(siteArea.siteID)) {
-        actions = [
-          this.editAction,
-          this.canCrudSiteArea ?
-            this.assignChargingStationsToSiteAreaAction :
-            this.viewChargingStationsOfSiteArea,
-          new TableMoreAction([
-            this.exportOCPPParamsAction,
-            openInMaps,
-            this.siteAreaGenerateQrCodeConnectorAction,
-            this.deleteAction,
-          ]).getActionDef(),
-        ];
-        if (this.isAssetComponentActive) {
-          actions.splice(2, 0, this.assignAssetsToSiteAreaAction);
-        }
+    const moreActions = new TableMoreAction([]);
+    if (siteArea.canUpdate) {
+      actions.push(this.editAction);
+      // flag to be added on   the backend for increased granularity
+      moreActions.addActionInMoreActions(this.exportOCPPParamsAction);
+      moreActions.addActionInMoreActions(this.siteAreaGenerateQrCodeConnectorAction);
```

Comment:
Are these always available when canUpdate is true? I'm not sure at the moment what the backend check should be based on.

Reply:
Yes, but it's only temporary. This is the way it was before the new auth concept implementation. Once the flag's in place it'll be driven by it.